src/box/lua/iproto.c

```diff
@@ -6,11 +6,30 @@
 
 #include "box/lua/iproto.h"
 
+#include "box/box.h"
 #include "box/iproto_constants.h"
 #include "box/iproto_features.h"
 
+#include "core/assoc.h"
+#include "core/fiber.h"
```

Comment:
You don't need `core/` prefix, it is implicitly included anyway.

Reply:
TBH, I don't like how we manage are `include`s: IMO, it's better to implicitly write the include path (e.g., do differentiate `core/fiber.h` from `lua/fiber.h`.)